cpu/cc3200/periph/gpio.c

```diff
@@ -62,7 +67,8 @@ static uint32_t ports[] = {
 /**
  * @brief   static callback memory
  */
-static gpio_isr_ctx_t isr_ctx[ARRAY_SIZE(ports)][GPIO_PINS_PER_PORT];
+static gpio_isr_ctx_t isr_ctx[sizeof(ports) / sizeof(uint32_t)]
+                             [GPIO_PINS_PER_PORT];
```

Comment:
I liked it better before (`ARRAY_SIZE(ports)`)

Reply:
me too... But for some reason while testing the gpio the ARRAY_SIZE macro was no longer defined. It may sound somewhat newbie but is there an ARRAY_SIZE definition somewhere or do I have to provide that myself.

Reply:
It's in [`kernel_defines.h`](https://github.com/RIOT-OS/RIOT/blob/master/core/include/kernel_defines.h) - although you might have to rebase to `master` to get it :wink:

Reply:
Makes sense! I will rebase to master and retest on hardware tomorrow. I probably told you thank you about a 100 times but really thanks for your support @benpicco.